config.yaml

```diff
@@ -9,6 +9,7 @@ mqtt:
     network: devices/%s/network
     logs: devices/%s/logs
     fingerprint: devices/%s/fingerprint
+    candump: devices/%s/protocol/canbus/dump
```

Comment:
@zakharenkodmytro do we need to add this to some cloudflare endpoint / worker?

Reply:
No need, I did not add anything to Cloudflare for other mqtt topics. It should work as it is.